index.html

```diff
@@ -5477,10 +5593,24 @@ <h3>Algorithm</h3>
                   <code>xsd:double</code> and its
                   <a>lexical form</a>
                   is a valid <code>xsd:integer</code> or <code>xsd:double</code>
-                  according [[XMLSCHEMA11-2]], set <var>converted value</var>
+                  according to [[XMLSCHEMA11-2]], and if it is a numeric value
+                  conformant to [[IEEE-754-2008]], then set <var>converted value</var>
                   to the result of converting the
                   <a>lexical form</a>
-                  to a JSON <a>number</a>.</li>
+                  to a JSON <a>number</a>.
+                </li>
+                <li>
+                  Otherwise, if <a>lexical form</a> expresses a numeric value
+                  outside of the ranges supported by [[IEEE-754-2008]], then set
+                  <var>converted value</var> to a JSON string literal containing
+                  said lexical form.
+                </li>
+                <li>
+                  Otherwise, if <a>lexical form</a> expresses a non-numeric value,
+                  such as <code>"+INF"^^xsd:double</code>, an
+                  <a data-link-for="JsonLdErrorCode">invalid JSON literal</a>
+                  error has been detected and processing is aborted.
+                </li>
```

Comment:
Any specific range examples should be careful to be informative, to not get ahead of XSD. Also, it will need to be consistent with JCS.

It might be that they way to do this is to just use JCS for serializing strings, and require that the result of serializing to JCS must result that is in the value space of xsd:integer or xsd:double, respectively.